### PR TITLE
Add how_to_book to SpaceOwner, and allow editing it inline from Space

### PR DESCRIPTION
**Pull request by DanielJackson-Oslo**
Sort of a WIP, as I think I should remodel the views for @Space, and also add the possiblity of adding common info to almost all rich text fields. 

Currently works like this:

SpaceOwner has a new :title

SpaceOwner also has a field :how_to_book that  is meant for information that is common to all spaces owned by that owner. 

Those fields are used throughout, and editable inline through @Space with `accepts_nested_attributes_for :space_owner`

### CODE REVIEW

File: app/views/spaces/show/_rich_text_with_space_owner_addendum.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<%
```
[the remark this answers is not in the document]
Comment (mathias234) [reply app/views/spaces/show/_rich_text_with_space_owner_addendum.html.erb:1]: The general rule is that you shouldn't do much logic in the view files, so either put them into helpers or the controller

File: app/helpers/spaces_helper.rb [hunk 2]
```diff
@@ -6,14 +6,30 @@ def edit_field(partial, form)
...
   end
 
+  def render_space_and_owner_field(space, field)
+    # rubocop:disable Style/IfUnlessModifier
+    field_in_space = (if space.respond_to?(field) && space.public_send(field).present?
+                        space.public_send(field)
+                      end)
+    field_in_space_owner = (if space.space_owner.respond_to?(field) && space.space_owner.public_send(field).present?
+                              space.space_owner.public_send(field)
+                            end)
```
Comment (mathias234) [inline app/helpers/spaces_helper.rb:24]: This looks wierd to me, should it probably be like this?
```rb
field_in_space = space.public_send(field) if space.respond_to?(field) && space.public_send(field).present?
field_in_space_owner = space.space_owner.public_send(field) if space.space_owner.respond_to?(field) && space.space_owner.public_send(field).present?
```

Comment (DanielJackson-Oslo) [reply app/helpers/spaces_helper.rb:24]: Rubocop didn't like the long lines, so I tried to break them up, but agreed that it looks weirder this way... I could, I guess, break it up like this:

```ruby
    owner = space.space_owner
    # Setting the fields only if they are available (respond_to) and they have any content (present?)
    field_in_space = (space.public_send(field) if space.respond_to?(field) && space.public_send(field).present?)
    field_in_space_owner = (owner.public_send(field) if owner.respond_to?(field) && owner.public_send(field).present?)

```

The paranthesis is needed to return nil as field_in_space, else ruby interprets it as "(field_in_space = foo) if bar", and  we end up with field_in_space possibly being undefined.

Comment (DanielJackson-Oslo) [reply app/helpers/spaces_helper.rb]: Tried fixing it like that, what do you think?

**Pull request closed without merging**